src/components/file/examples/file-no-dropzone.tsx

```diff
@@ -0,0 +1,79 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+/**
+ * No dropzone visualization
+ *
+ * You can hide the dropzone visualization by setting `visualize-Dropzone` property
+ * to `false`.
+ * :::note
+ * - Even if you hide the visualization, the component will still support
+ * drag and drop. But only the visualization of the dropzone will dissapear.
+ * - A `readonly` file picker does not show the dropzone.
```

Comment:
It is good to mention that `readonly` disables adding and removing files.

Reply:
Hmm… I'm not sure, but I think I disagree. For one thing, I think it ought to be fairly self-evident that files can't be added and removed when the component is `readonly`. But, even if that _does_ need explaining, I think it should be explained in the docs for `readonly`, or in an example specifically for `readonly`. I don't think that it should be explained here, in the example for `dropzone`.

Reply:
Me neither. I think the concept comes with `readonly`. So I wouldn't explain it.